src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -77,6 +79,60 @@ internal static ParseResult ParseScript(string script, ILanguageSettings loc = n
             return new ParseResult(parsetree, errors, errors?.Any() ?? false, parser._comments, parser._before, parser._after);
         }
 
+        public static ParseFormulasResult ParseFormulasScript(string script, ILanguageSettings loc = null, Flags flags = Flags.None)
+        {
+            Contracts.AssertValue(script);
+            Contracts.AssertValueOrNull(loc);
+
+            flags = Flags.NamedFormulas | flags;
```

Comment:
The user could pass in EnableExpressionChaining as a flag right? It might be a surprise to the user that the NamedFormulas flag essentially removes the chaining behavior.

Reply:
That's a good point. I'm not sure if there's a good way to make it clear that they're mutually exclusive flags, although at the least we should add a comment in the enum calling that out

Reply:
Why even accept flags to this function at all?

Reply:
Good point. ReplaceableExpressions was a "c-style macro" feature that never shipped thankfully, and we're never allowing chaining in these expressions. 
We shouldn't allow the user to pass flags at all

Reply:
I removed the ability to add flags, and added a comment to the named formulas flag.